src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  --head-marg: 50px;
+  --max-header-wight: 1200px;
+  --min-header-wight: 1024px;
+  --special-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for `Roboto` to ensure text is displayed with a default sans-serif font if `Roboto` fails to load.